src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -473,7 +473,7 @@ public function preprocessEditorialLabel($cdbid, $settings)
         $editorialLabelText = 'UiT-tip van ';
         $label = $editorialLabelText . implode(", ", $publishers);
 
-        if (count($publishers) > 1 ) {
+        if (count($publishers) > 1) {
```

Comment:
👍 
(Noticed this in a previous commit, good that you spotted it yourself as well)